app/main.py

```diff
@@ -2,0 +1,12 @@
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    filename_in = parts[1]
```

Comment:
Consider using more descriptive variable names, such as `source_file_name` and `destination_file_name`, to improve code readability. This aligns with the checklist's recommendation for using descriptive and correct variable names.